dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -70,7 +63,7 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::RegisterF
     auto lock = m_lock.lock_exclusive();
     THROW_HR_IF(WPN_E_PLATFORM_UNAVAILABLE, m_shutdown);
 
-    const std::wstring appIdentifier = GetAppIdentifier(processName);
+    const std::wstring appIdentifier{ BuildAppIdentifier(processName) };
```

Comment:
Can you add a comment over both GetAppIdentifier/BuildAppIdentifier saying that each function assume the caller is under a lock? Then both functions don't need locks.